src/autopas/containers/linkedCells/traversals/LCC08CellHandlerUtility.h

```diff
@@ -0,0 +1,150 @@
+/**
+ * @file LCC08CellHandlerUtility.h
+ * @author J. Schuhmacher
+ * @date 11.07.2024
+ */
+
+#pragma once
+
+#include <array>
+#include <tuple>
+#include <type_traits>
+#include <utility>
+#include <vector>
+
+#include "autopas/utils/ArrayMath.h"
+#include "autopas/utils/ArrayUtils.h"
+#include "autopas/utils/Math.h"
+#include "autopas/utils/ThreeDimensionalMapping.h"
+
+namespace autopas::internal {
+
+/**
+ * Type Alias for the C08 base step:
+ * A triplet consisting of:
+ *  - offset of first cell
+ *  - offset of second cell
+ *  - sorting direction a.k.a. the normalized vector between cell1 and cell2 connecting their centers. This is used
+ *      in the {@link autopas::internal::CellFunctor} for AoS processing of Cell Pairs, ultimatley in
+ *      {@link autopas::SortedCellView} for building a projection order of particles to early stop the processing.
+ */
+using OffsetPairSorting = std::tuple<unsigned long, unsigned long, std::array<double, 3>>;
+
+/**
+ * A pair consisting of:
+ *  - offset of first cell
+ *  - offset of second cell
+ */
+using OffsetPair = std::pair<unsigned long, unsigned long>;
```

Comment:
for the sake of readability you could consider replacing the tuple and pair with structs that have properly named members.

Reply:
Good point. I would put that on hold until the sorting-offset-debate is settled because `std::pair` has one advantage: It provides by-default `==` and `<` operators. So, depending on, if we employ some sorting on the offset pairs. It might come in handy.